packages/python-runner/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@scramjet/python-runner",
+  "version": "0.17.0",
+  "description": "This package is part of Scramjet Transform Hub. The package executes user code inside a python process communicating with Transform Hub.",
+  "main": "./runner.py",
+  "scripts": {
+    "preinstall": "pip install --upgrade -r requirements.txt --target dist; cp -r ../../python/scramjet-framework/scramjet/ dist/",
```

Comment:
i'm not sure here
if i will do `npm i @scramjet/python-runner` in empty folder it will crash, right?

Reply:
@patuwwy please check again.